addons/fluxcd/definitions/helm-release-def.cue

```diff
@@ -48,49 +50,52 @@ helm: {
 }
 
 template: {
-	output: {
-		apiVersion: "source.toolkit.fluxcd.io/v1beta2"
-		metadata: {
-			name: context.name
-		}
-		if parameter.repoType == "git" {
-			kind: "GitRepository"
-			spec: {
-				url: parameter.url
-				if parameter.git.branch != _|_ {
-					ref: branch: parameter.git.branch
+	outputs: {
```

Comment:
why make the source to be outputs?

Reply:
The Error `Object 'Kind' is missing in '{...}'` always occurs when I put source in output.
Maybe because I add `if parameter.sourceName == _|_` expression so the output field could be nil I'm not quite sure about this.

Reply:
I see, yes you're right here, the output must have object while if you have sourceName the source object will be ommitted.